packages/component-library/src/components/table-and-list/mt-pagination/mt-pagination.spec.ts

```diff
@@ -115,5 +121,29 @@ describe("mt-pagination", () => {
 
       expect(wrapper.emitted()["change-current-page"][0]).toStrictEqual([7]);
     });
+
+    it("should disable the next page and last page button when total page value is zero", async () => {
+      render(MtPagination, {
+        props: {
+          currentPage: 1,
+          totalItems: 0,
+          limit: 25,
+        },
+        global: {
+          mocks: {
+            $t: mockT,
+          },
+        },
+      });
+
+      const nextPageButton = screen.getByTestId("mt-pagination-next-page-button");
+      const lastPageButton = screen.getByTestId("mt-pagination-last-page-button");
```

Comment:
Please retrieve the elements with `[getByRole](https://testing-library.com/docs/queries/byrole/)` method. This makes sure that the components is accessible and actually uses buttons instead of divs to make the component interactive.

Reply:
I updated the test now. However, the current button does not have a specific role. So Please check if my approach is correct.